dashboard/app/models/stage.rb

```diff
@@ -2,14 +2,15 @@
 #
 # Table name: stages
 #
-#  id            :integer          not null, primary key
-#  name          :string(255)      not null
-#  position      :integer
-#  script_id     :integer          not null
-#  created_at    :datetime
-#  updated_at    :datetime
-#  flex_category :string(255)
-#  lockable      :boolean
+#  id                :integer          not null, primary key
+#  name              :string(255)      not null
+#  absolute_position :integer
+#  script_id         :integer          not null
+#  created_at        :datetime
+#  updated_at        :datetime
+#  flex_category     :string(255)
+#  lockable          :boolean
+#  relative_position :string(255)
```

Comment:
What would be the best place to document something like this in your opinion? Maybe above the line `acts_as_list scope: :script, column: :absolute_position`